docs/api.rst

```diff
@@ -0,0 +1,12 @@
+
+.. If you created a package, create one automodule per module in the package.
+
+.. If your library file(s) are nested in a directory (e.g. /adafruit_foo/foo.py)
+.. use this format as the module name: "adafruit_foo.foo"
+
+.. automodule:: adafruit_ra8875.ra8875
+   :members:
+
+.. automodule:: adafruit_ra8875.registers
+   :members:
```

Comment:
`registers` seems to be for internal use only? If they aren't going to be exposed to the user, I don't really see a need to list them in the documentation. _May_ add a layer of confusion. I can definitely be overruled on this, though. 😄

Reply:
That is correct. Internal use only. I just included them in a separate file because they cluttered up the code.

Reply:
Oh I get it, should I remove the automodule for that then?

Reply:
I went ahead and removed the automodule.